aii-opennebula/src/main/resources/tests/profiles/vm.pan

```diff
@@ -97,7 +105,8 @@ bind "/system/opennebula" = opennebula_vmtemplate;
 prefix "/system/opennebula";
 "vnet" = nlist(
     "eth0", "br100",
-    "eth1", "br101");
+    "eth1", "br101",
+    "eth2", "br101");
 
```

Comment:
yes with different pros and cons @samary @StephaneGerardVUB do you have any preference about these options? use an ignoremac option per interface or to use it as global boolean